docs/data/base/components/slider/slider.md

```diff
@@ -9,20 +9,111 @@ packageName: '@mui/base'
 
 # Unstyled slider
 
-<p class="description">The <code>SliderUnstyled</code> component lets users make selections from a range of values along a horizontal or vertical bar.</p>
+<p class="description">A slider is a UI element that lets users select a single value or a range of values along a bar.
+</p>
 
-Users may need to select a single value or a range of values on a slider.
-They are ideal for interface controls that benefit from a visual representation of adjustable content, such as volume or brightness settings, or for applying image filters.
+## Introduction
+
+The `SliderUnstyled` component lets users make selections from a range of values along a horizontal or vertical bar.
+
+Sliders are ideal for interface controls that benefit from a visual representation of adjustable content, such as volume or brightness settings, or for applying image filters such as gradients or saturation.
 
 {{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
 
-```js
+## Component
+
+### Anatomy
+
+After [installation](/base/getting-started/installation/), you can start building with this component using the following basic elements:
+
+```jsx
 import SliderUnstyled from '@mui/base/SliderUnstyled';
+
+export default function MyApp() {
+  return <SliderUnstyled />;
+}
 ```
 
+### Basic usage
+
+The following demo shows how to create and style two basic sliders.
+Notice that both are set to a default value of 10 with the `defaultValue` prop, and the the second slider cannot be adjusted due to the `disabled` prop:
+
 {{"demo": "UnstyledSlider.js", "defaultCodeOpen": false}}
 
-## Discrete sliders
+### Component slots
+
+The `SliderUnstyled` component is composed of a root `<span>` that houses three interior `<span>` elements:
+
+- rail: the full length of the slider
+- track: the section of the slider that's active
+- thumb: the button that the user moves across the slider
+
+```html
+<span class="MuiSlider-root">
```

Comment:
Food for thought, there are more slots here, like: `Mark`, `MarkLabel`, `ValueLabel`, and some of them are shown conditionally. I think it's worth adding them in here too.

Reply:
How should we go about documenting the conditional slots? This code snippet could get really confusing if I try to add them all in. Do you think it would be enough to list them? Should we take a different approach altogether to documenting the slots?